src/main/java/org/fundacionjala/at15/katas/bankocr/sergio/NumberDef.java

```diff
@@ -0,0 +1,79 @@
+package org.fundacionjala.at15.katas.bankocr.sergio;
+
+public class NumberDef {
+    private MatrixArch matrix;
+
+    public NumberDef(MatrixArch currentMatrix) {
+        this.matrix = currentMatrix;
+
+    }
+
+    public String loneDigit(int start) {
```

Comment:
Maybe the logic of the conditions in this method can be improved so we don't have too many branches to test